StackOverflow/Controller/QuestionsCell.h

```diff
@@ -0,0 +1,18 @@
+//
+//  QuestionsCell.h
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface QuestionsCell : UITableViewCell
+
+@property (weak, nonatomic) IBOutlet UIImageView *acceptedAnswerImage;
```

Comment:
Naming is very important, and as a code reviewer I'll always spend a considerable amount of time on naming. So in this case, the name is incorrect. It's not an image, it's an imageView and therefore this should be named acceptedAnswerImageView.

Reply:
Thank you, I will correct this